src/GraphQL/ParseGraphQLSchema.js

```diff
@@ -203,7 +203,6 @@ class ParseGraphQLSchema {
 
     if (this.graphQLCustomTypeDefs) {
       schemaDirectives.load(this);
-
       if (typeof this.graphQLCustomTypeDefs.getTypeMap === 'function') {
         // In following code we use underscore attr to avoid js var un ref
```

Comment:
Could you clarify what "js var un ref" means?

Reply:
i updated the comment